robot-logic/src/main/java/logic/springconfig/ResetRestController.java

```diff
@@ -38,4 +42,12 @@ public ResetResponse reset() {
         LocalDateTime lastDate = TIME_MANAGER.getLastRun();
         return new ResetResponse(lastDate.getHour(), lastDate.getMinute(), lastDate.getSecond(), canRun);
     }
+
+
+    @ControllerAdvice
+    public static class JsonpAdvice extends AbstractJsonpResponseBodyAdvice {
+        public JsonpAdvice () {
+            super("callback");
+        }
+    }
 }
```

Comment:
Not clear, what @ControllerAdvice does here.Maybe making a Javadoc for this, will be helpful.